### PR TITLE
Fix-2758

### PR DESCRIPTION
**Pull request by ramkiperiy**
Fixed issues 2758, changed fixture to be at class level.
Commented ceph and resource checks which is increasing the scale execution time

Signed-off-by: Ramakrishnan <rperiyas@redhat.com>

### CODE REVIEW

**Comment by ramkiperiy** [comment]
Fixes issue https://github.com/red-hat-storage/ocs-ci/issues/2758

**Comment by ramkiperiy** [comment]
Testing it locally, can't execute this tests in jenkins due to memory allocation problem.

File: tests/e2e/scale/test_ceph_pod_respin_in_scaled_cluster.py [hunk 1]
```diff
@@ -25,9 +25,19 @@ def fioscale(request):
         node_selector=constants.SCALE_NODE_SELECTOR
     )
     fioscale.create_scale_pods(
-        scale_count=1500, pods_per_iter=10, io_runtime=36000,
+        scale_count=350, pods_per_iter=10, io_runtime=36000,
```
Comment (keesturam) [inline tests/e2e/scale/test_ceph_pod_respin_in_scaled_cluster.py:28]: @ramkiperiy We used to run IO on 1500 pods for 10 hours earlier and now we run,
IOs on 350 pods for 10 hours, followed by the creation of 350 pods without IOs and then 400 pods with ~1 hour and 400 pods without IO. 

We'll simply use 400 pods for each iteration and run IOs for 18000 seconds alternatively. IMO, It is okay to cross 1500 pods. (we anyway don't run IOs on all of them)

Also, is there any reason we run IO for such a long time?

**Review by keesturam — CHANGES_REQUESTED** [review]

**Comment by ramkiperiy** [comment]
Planning to implement kube jobs to scale pvc+pod, so it then holding on merging this PR.
Will create new PR for changing the fixture scope to "Class"

**Comment by ramkiperiy** [comment]
Closing this PR since entire script needs re-work to use kube_job or other python method which doesn't use more Object creations.

**Pull request closed without merging**